### PR TITLE
Allow users to navigate the Create Project form out of order

### PR DESCRIPTION
**Pull request by sdevalapurkar**
# Overview

## This PR contains the following changes

- Allow users to navigate the Create Project form out of order

## This PR contains the following types of changes

- [x] New feature (change which adds functionality)
- [x] Enhancement (improvements to existing functionality)

## How Has This Been Tested?

- Locally

## Screenshots

image

### CODE REVIEW

File: app/src/features/projects/CreateProjectPage.tsx [hunk 1]
```diff
@@ -133,6 +134,9 @@ const CreateProjectPage: React.FC = () => {
   // Tracks the active step #
   const [activeStep, setActiveStep] = useState(0);
 
+  // Tracks if sections of the form have been completed
+  const [formsComplete, setFormsComplete] = useState([false, true, false, false, false, true, true, true, true]);
```
Comment (anissa-agahchen) [inline app/src/features/projects/CreateProjectPage.tsx]: maybe replace complete with valid to reflect that the form, as complete can imply something else

**Review by anissa-agahchen — CHANGES_REQUESTED** [review]
When you open a saved draft, the forms should also open that the first incomplete section.

File: app/src/features/projects/CreateProjectPage.tsx (hunk 1 above)
Comment (sdevalapurkar) [reply app/src/features/projects/CreateProjectPage.tsx]: `formStepsValid`

**Comment by sdevalapurkar** [comment]
> When you open a saved draft, the forms should also open that the first incomplete section.

will handle in different ticket as not part of acceptance criteria and also requirement is still unclear

**Review by anissa-agahchen — APPROVED** [review]

**Pull request merged**